lib/enerscore/response_parser.rb

```diff
@@ -4,24 +4,23 @@ class ResponseParser
 
     def initialize(response)
       @response = response
-      if @response == :network_timeout
+
+      case @response
+      when :network_timeout
```

Comment:
Additionally, one other refactor could be just returning the status to the top of the case statement instead of assigning it at each step.

```
@status = case @response
when :success
   @parsed_response = response_object(response)
   has_results?(@parsed_response) ? :success : :no_results
...